pkgs/development/python-modules/flask-expects-json/default.nix

```
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+, jsonschema
+}:
+
+buildPythonPackage rec {
+  pname = "flask-expects-json";
+  version = "1.7.0";
```

```suggestion
  version = "1.7.0";
  format = "setuptools";
```

We try to make format a required attribute.